app/uk.gov.hmrc.helptosavestridefrontend/views/partials/h1.scala.html

```diff
@@ -0,0 +1,23 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this()
+
+@(key: String, classes: Option[String] = None, id: Option[String] = None)(implicit msgs: Messages)
+
+@cls = @{"govuk-heading-l ".concat(classes.getOrElse(""))}
+
+<h1 class="@cls">@Html(msgs(key))</h1>
```

Comment:
I don't think this should be wrapped in Html(), here you are purely rendering a message from a key, messages for H1 content should not have HTML within the message - rare cases to justify using HTML markup in messages include mid-sentence occurrence of links, visually hidden text, abbreviation tags and strong elements - none of these are used in H1 content